fuzzytm 0.3.0 (GPL-2.0-only) has 2 incompatibilities in [tree_created]:
fuzzytm 
  pyfume 0.2.5 [fuzzytm ()]
    scipy 1.7.3 [fuzzytm ()] [pyfume ()] [simpful (>=1.0.0)]
    numpy 1.21.5 [fuzzytm ()] [pyfume ()] [scipy (<1.23.0,>=1.16.5)] [pandas (>=1.17.3)] [simpful (>=1.12.0)]
    fst-pso 1.8.1 [pyfume ()]
    simpful 2.5.0 (INCOMPATIBLE! HAS GPL-3.0-only) [pyfume ()]
      requests 2.26.0 (INCOMPATIBLE! HAS Apache-2.0) [simpful ()]
        idna 3.3 [requests (<4,>=2.5)]
        charset-normalizer 2.0.9 [requests (~=2.0.0)]
        certifi 2021.10.8 [requests (>=2017.4.17)]
        urllib3 1.26.7 [requests (<1.27,>=1.21.1)]
  scipy 1.7.3 [fuzzytm ()] [pyfume ()] [simpful (>=1.0.0)]
    numpy 1.21.5 [fuzzytm ()] [pyfume ()] [scipy (<1.23.0,>=1.16.5)] [pandas (>=1.17.3)] [simpful (>=1.12.0)]
  sparsesvd 0.2.2 [fuzzytm ()]
  pandas 1.3.5 [fuzzytm ()]
    numpy 1.21.5 [fuzzytm ()] [pyfume ()] [scipy (<1.23.0,>=1.16.5)] [pandas (>=1.17.3)] [simpful (>=1.12.0)]
    pytz 2021.3 [pandas (>=2017.3)]
    python-dateutil 2.8.2 [pandas (>=2.7.3)]
      six 1.16.0 [python-dateutil (>=1.5)]
  numpy 1.21.5 [fuzzytm ()] [pyfume ()] [scipy (<1.23.0,>=1.16.5)] [pandas (>=1.17.3)] [simpful (>=1.12.0)]
